src/ModelContextProtocol.AspNetCore/Auth/McpAuthorizationExtensions.cs

```diff
@@ -0,0 +1,28 @@
+using Microsoft.AspNetCore.Authentication;
+using Microsoft.Extensions.DependencyInjection.Extensions;
+
+namespace ModelContextProtocol.AspNetCore.Auth;
```

Comment:
```suggestion
namespace Microsoft.Extensions.DependencyInjection;
```

This is the pattern we follow for AuthenticationBuilder extension methods. It's a bit weird since even AuthenticationBuilder doesn't live there, but we're pretty consistent with this, and it reduces the number of usings a typically AP.NET Core app needs.